.agent/skills/drn-domain-design/SKILL.md

```
@@ -1,421 +1,327 @@
+description: Domain-Driven Design implementation patterns - Entity design with Source-Known IDs (SourceKnownEntity, AggregateRoot), repository contracts (ISourceKnownRepository), domain events, EF Core configuration, DTO mapping rules, and implementation templates. Essential for domain modeling and data access. Keywords: ddd, entity-design, aggregate-root, source-known-id, repository, domain-events, ef-core-configuration, dto-mapping, entity-type, identity
+last-updated: 2026-02-15
+difficulty: advanced
 ---
 
-# Domain Design Skill (DRN.Framework)
+# Domain Design Patterns
 
-> Type-safe domain modeling with Source-Known IDs, repository patterns, and EF Core conventions.
+> Entity design, repository implementation, and DDD patterns for DRN applications.
 
 ## When to Apply
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)